lib/l10n_utils/__init__.py

```diff
@@ -55,55 +57,31 @@ def render(request, template, context=None, **kwargs):
     context['langfile'] = get_lang_path(template)
     context['template_source_url'] = template_source_url(template)
 
-    # Get the available translation list of the current page
-    context.setdefault('translations', {})
-    context['translations'].update(translations_for_template(template))
-
-    # Look for localized template if not default lang.
-    if hasattr(request, 'locale') and request.locale != settings.LANGUAGE_CODE:
-
+    # if `locales` is given use it as the full list of active translations
```

Comment:
Does this over-ride the language files then?

Reply:
In this case yes. In this way you can have a page (e.g. `/about/legal/impressum/`) that is not available in en-US.